example/admin/config.go

```diff
@@ -106,7 +106,7 @@ func NewConfig() Config {
 	// perm.Verbose = true
 	b.Permission(
 		perm.New().Policies(
-			perm.PolicyFor(perm.Anybody).WhoAre(perm.Allowed).ToDo(presets.PermCreate, presets.PermUpdate, presets.PermDelete, presets.PermGet, presets.PermList).On("*:roles:*", "*:users:*"),
+			perm.PolicyFor(perm.Anybody).WhoAre(perm.Allowed).ToDo(perm.Anything).On("*"),
```

Comment:
the result of this is anybody has permission to any resource. (it will ignore other perm definitions)